crates/driver/src/infra/persistence/mod.rs

```diff
@@ -0,0 +1,70 @@
+use {
+    crate::{domain::competition::auction::Id, infra::solver::Config},
+    serde::Deserialize,
+    std::sync::Arc,
+    tracing::Instrument,
+};
+
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub struct S3 {
+    /// The s3_instance_upload_* arguments configure how auction instances
+    /// should be uploaded to AWS S3.
+    /// They must either all be set or all not set.
```

Comment:
`s3_instance_upload_*` is no longer a thing.
> They must either all be set or all not set.

Since we now use a config file instead of env variables this is finally enforced by the deserialization logic so we don't have to mention it anymore.

And finally this comment doesn't really explain what a bucket is.